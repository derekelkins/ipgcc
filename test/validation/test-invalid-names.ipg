S -> _ipg_startsWith;

_ipg_startsWith -> { this = . };

A -> { these = . };
B -> { _ipg_start = . };
C -> { _ipg_end = . };
