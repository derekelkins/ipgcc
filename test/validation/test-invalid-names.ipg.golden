_ipg_startsWith can't be the name of a Rule
Rule A illegally assigns to these
Rule B illegally assigns to _ipg_start
Rule C illegally assigns to _ipg_end
Rule _ipg_startsWith illegally assigns to this